index.yaml

```diff
@@ -4274,6 +4274,37 @@ entries:
     - assets/longhorn-crd/longhorn-crd-1.0.200.tgz
     version: 1.0.200
   neuvector:
+  - annotations:
+      catalog.cattle.io/auto-install: neuvector-crd=match
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: NeuVector
+      catalog.cattle.io/kube-version: '>=1.18.0-0 < 1.29.0-0'
```

Comment:
I don't know much about neuvector so forgive me if I am wrong. 

According to the patterns we use on Mapps and that I see in other charts on Rancher 2.7 version I think we only support until 1.27 kubernetes version.

Reply:
@nicholasSUSE NeuVector supports k8s 1.28 so updated the annotation. I tested the chart on downstream cluster with k8s 1.28.6

Reply:
@nicholasSUSE This chart might be used outside Rancher Manager. We offer all the charts via charts.rancher.io so users can use them via `helm CLI`. It's up to the feature chart teams to say in which k8s versions the charts might be installed